packages/core/src/Stores/client-store.js

```diff
@@ -2246,6 +2252,26 @@ export default class ClientStore extends BaseStore {
             this.is_populating_account_list = true;
             const authorize_response = await BinarySocket.authorize(is_client_logging_in);
 
+            if (is_social_signup_provider) {
```

Comment:
anyway that we can do this without altering the stores? as you may know the plan for the stores.

Reply:
Need to introduce side effects in the component then and since all the logic is there for the signup flow, I thought it would be okay to put this here, if you think it's best to  do this without altering stores at all let me know, I'll refactor

Reply:
please check the possibility of it. if its not require lots of changes then we better go that way.

Reply:
@ali-hosseini-deriv this flow is from the continuation of the signup and this is triggering when we signup from our social signup provider, after checking, I think this would be the best place to put this as of now